test/aws/default/verify/controls/aws_kms_key.rb

```diff
@@ -0,0 +1,51 @@
+fixtures = {}
+[
+  'kms_key_disabled_arn',  
+  'kms_key_enabled_arn',
+  'kms_key_recall_hit_arn',
+  'kms_key_enabled_key_id',
+  'kms_key_enabled_key_description'
+].each do |fixture_name|
+  fixtures[fixture_name] = attribute(
+  fixture_name,
+  default: "default.#{fixture_name}",
+  description: 'See ../build/kms.tf',
+  )
+end
+
+control "aws_kms_key recall" do
+  describe aws_kms_key(fixtures['kms_key_recall_hit_arn']) do
+    it { should exist}
+  end
+  describe aws_kms_key(key_id: fixtures['kms_key_recall_hit_arn']) do
```

Comment:
The API doesn't care if you send an ARN or an ID, so I think this works, but you should probably be sending the key_id here rather than the ARN.